packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -275,6 +451,8 @@ def _verify_tx(self, tx_hash: str) -> Generator[None, None, ContractApiMessage]:
             self.synchronized_data.most_voted_tx_hash
         )
 
+        chain_id = self.synchronized_data.get_chain_id(self.params.default_chain_id)
```

Comment:
I think it makes more sense to include this in the tx data. 
Later we can pass it like this:
https://github.com/valory-xyz/agent-academy-2/blob/main/packages/valory/skills/keep3r_job_abci/behaviours.py#L602

Reply:
It will mean also changes on the payload tools:
https://github.com/valory-xyz/agent-academy-2/blob/main/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L91